backend/config/initializers/form_builder.rb

```diff
@@ -14,7 +14,7 @@ def error_message_on(method, options = {})
 
   def field_hint(method, options = {})
     title = options.delete(:title) || @object.class.human_attribute_name(method)
-    text = options.delete(:text) || I18n.t(method, scope: [:spree, :hints, @object.class.model_name.i18n_key], **options)
+    text = options.delete(:text) || I18n.t(method, scope: [@object.class.module_parent.to_s.underscore.to_sym, :hints, @object.class.model_name.i18n_key], **options)
```

Comment:
Hmm, not about this approach

What about allowing to pass the translation in the `options` hash?

Reply:
I just saw that the `text` is already passed in the options. What about adding your translation in the calling code?